ribs/archives/_archive_base.py

```diff
@@ -434,22 +471,23 @@ def add(self,
         index_batch = self.index_of(measures_batch)
 
         # Copy old objectives since we will be modifying the objectives storage.
-        old_objective_batch = np.copy(self._objective_arr[index_batch])
+        old_objective_arr = np.copy(self._objective_arr[index_batch])
```

Comment:
I called these `batch` because they correspond to the solutions in the current batch -- can you switch them from `arr` to `batch`?

Reply:
Also, changed `old_threshold_arr` to `old_threshold_batch`